spec/models/user_spec.rb

```diff
@@ -55,7 +55,7 @@
     expect(user.first_name).to eq "Bob"
 
     user = FactoryBot.create :user, last_name: " Jo nes "
-    expect(user.last_name).to eq "Jo nes"
+    expect(user.last_name).to eq "Jones"
```

Comment:
I'm not sure how this passing before, so updated it. This is supposed to be stripping white space from the names.

Reply:
nevermind - figured it out :)